rep11.md

```diff
@@ -0,0 +1,183 @@
+Challenger->>Challenger: unsubscribe from "stat/<challangee-address>/POPCHALLENGERESULT"
+end
+
+Challenger->>Challenger: Create PoP Result: ReportPopResult with the challenge details.
+Challenger->>Challenger: set Challenge.Success = (CID` == CID)
+Challenger->>Planetmint-Node: notarize and broadcast PoP Result
+
+```
+
+### Rewards
+The following paragraphs give an overview of how rewards are created and managed. The following stages are processed until someone can access their RDDL rewards:
+
+* Reward Calculation
+* Clearance Process: re-issuance & distribution
+
+Before introducing these aspects, the generic staging and claiming of the rewards is addressed.
+
+#### Staging the Rewards
+RDDL tokens are issued as the final rewards of a PoP to a PoP-related multi-sig wallet(**PoP wallet**) on the Liquid network.
+
+Rewarded PoP participants can claim their rewards from this wallet. Tokens to claim RDDL from the PoP-Wallet are minted once a day by a clearance process on Planetmint. The minted tokens representing claims are named **cRDDL** (claimable RDDL).
+
+The block proposing validator verifying the ReportPoPResult message from the challenger mints **stagedCRDDL** following the reward calculation. The resulting stagedCRDDL tokens are converted to cRDDL by the clearance process.
+
+> PLMNT and STAKE are the only tradable tokens on the Planetmint Chain. stagedCRDDL and cRDDL cannot be traded but redeemed as RDDL tokens on Liquid. RDDL of course is tradable on Liquid. 
+> The configuration and voting proposal for this behaviour can be found at [voting-proposal.json](/data/REP-11/voting-proposal.json).
+
+
+#### Claiming RDDLS
+Details are to be defined. The claiming of RDDL will be issued by a Planetmint message. That message will burn cRDDL and transfer RDDL tokens from the PoP wallet to the specified Liquid address.
+
+#### Reward Calculation
+The network waits for the challenger to respond with the ReportPopResult for two PoP epochs (2 times 2 minutes). The rewards for the PoP are then calculated depending on the outcome of the PoP.
+
+```mermaid
+flowchart TD
+    X[After sending out the InitPoP]--> |Enter waiting state| Y
+    Y{Waiting for PoP Result}-->|No PoP Result received within the duration of 2 PoP epochs| J
+    Y-->|On PoP Result for the acting Validator| B{Is the message's creator the challenger?}
+    B -->|Created by Challenger| C{\n Has the challenge been solved?}
+    C -->|Challenge solved!| D[mint Staged RDDLclaims: 2% challenger,  6% challengee]
+
+    B -->|Not created by the challenger| J[8% to the DAO due to no trust in the PoP result.]
+    C -->|Challenge not solved| E[mint Staged RDDLclaims: 2% challenger,  6% DAO]
+```
+
+#### Clearance Process
+
+Reducing transaction fees on Liquid is the purpose of the clearance process. The clearance process is executed once a day. It contains two steps: the re-issuance and distribution of the RDDL tokens to the DAO, PoP and investor wallets.
+##### Re-issuance
```

Comment:
Here it is `re-issuance` and in line 179 it is reissuance. Did you do that on purpose? If not please stick to one way of writing it...

Reply:
I suggest creating a new ticket to get this changed on docs.rddl.io and the reps repository.